src/Logging/DataQueue.cpp

```diff
@@ -24,7 +24,7 @@ String DataQueue::publish(String event, PublishFlags flag1, PublishFlags flag2)
 	_publishQueue->publish(event, payload, flag1, flag2);
 	_writerRefresh();
 
-	DEBUG_SERIAL_LN("Number of events Queued: " + String(_publishQueue->getNumEvents()));
```

Comment:
I was having some issues with including the settings.h file so I commented these out. I think they should be turned into getters instead, and we should do all debugging in the main or vehicle files.

Reply:
What was the issue you were running into? I tried a minimal build with this line commented out and `settings.h` included and didn't come across any issues.

Reply:
It looks like it's now compiling fine, I think I was in makefile hell yesterday and it was a separate issue altogether. I'll revert this.